lib/Backend/BackwardPass.cpp

```diff
@@ -6755,7 +6755,7 @@ BackwardPass::ReverseCopyProp(IR::Instr *instr)
     }
 
     // The fast-path for these doesn't handle dst == src.
-    // REVIEW: I believe the fast-path for LdElemI_A has been fixed... Nope, still broken for "i = A[i]" for �prejit
+    // REVIEW: I believe the fast-path for LdElemI_A has been fixed... Nope, still broken for "i = A[i]" for �prejit
```

Comment:
Thanks @dilijev for reviewing! Looks some of these were due to Unicode characters in comments. I noticed them previously but forgot to update this time. I'll check and fix.